styleguides/react/core.md

```diff
@@ -0,0 +1,3 @@
+# React Standards and Style Guide
+
+TODO
```

Comment:
Perhaps direct people to the JavaScript standards?

Reply:
Done.